src/parents.js

```diff
@@ -43,5 +45,34 @@ export function set (node, parent, { force } = {}) {
  * @returns {object | undefined} The parent node, or undefined if the node has no parent
  */
 export function get (node) {
+	const {parent} = parentMap.get(node) ?? {};
+	return parent;
+}
+
+/**
+ * Get the parent node and metadata for a node.
+ * @param {object} node
+ * @returns {object | undefined} An object containing the parent node and the property name of the child node in the parent
+ */
+export function getWithMetadata (node) {
 	return parentMap.get(node);
 }
```

Comment:
I thought it would be nice to keep the `get` function simple -- most of the time I think users would only want the parent, and not care about the metadata, so I thought it would be most useful to have a second function that returns all the useful metadata that we need for other functions. What do you think of this @LeaVerou ?

Reply:
I think we need to look at the use cases. Can you collect a few from within vastly and Mavo?

Reply:
There's only a couple where parents are being accessed currently. Vastly does it in `closest` to walk up the tree, and Mavo does a similar thing in MavoScript to walk up a tree as well. Maybe having one function `get` that returns everything keeps it simplest, what do you think?

Reply:
Could you post some links to these uses so I don't have to go hunting them down?

Reply:
Vastly: https://github.com/mavoweb/vastly/blob/main/src/closest.js#L8
Mavo: https://github.com/mavoweb/mavo/blob/d68e709d1c2817704c8cab45046ac501f71f8937/src/mavoscript.js#L639
https://github.com/mavoweb/mavo/blob/d68e709d1c2817704c8cab45046ac501f71f8937/src/mavoscript.js#L829

Reply:
These seem to all be the same pattern. Ok, let’s go with `parents.get()` returning the parent, with another function returning the whole object. 

Name ideas: `parents.getPath()`, `parents.path()` (pretty sure there's something better though)

Reply:
Sounds good. Not sure what the best name would be. Do we want to include `path` in the name if we might add a separate `path` helper?

How about something like `parents.relation`, since the metadata returned details the relationship (how you get from parent to child)? That's the only good idea I thought of aside from simple ones like `parents.info`, `parents.metadata`, `parents.details`, but none of those fit super well.

Reply:
I like `parents.info()` or `parents.details()`. `parents.metadata()` is a bit long (though `parents.meta()` may be ok?). Pick the one you like and let's go with it, it's not like we have such a userbase rn that API changes are hard 😁  

Alternatively we could pass an argument to `get()` to give us the object instead of just the parent, but it's generally an [API design antipattern](https://w3ctag.github.io/design-principles/#overloading) to get entirely different types back depending on arguments.

Reply:
Pushed rename!